### PR TITLE
fix: Just reducing error log spam a bit

### PR DESCRIPTION
**Pull request by dkershner6**

### CODE REVIEW

File: packages/open-next/src/adapters/cache.ts [hunk 1]
```diff
@@ -490,7 +490,7 @@ export default class S3Cache {
       );
       return result;
     } catch (e) {
-      error("This error can usually be ignored : ", e);
+      warn("This error can usually be ignored : ", e);
```
Comment (khuezy) [inline packages/open-next/src/adapters/cache.ts:493]: How often are you seeing this error? If it's usually ignorable, then should we just use `debug` or omit the log altogether?

Comment (dkershner6) [reply packages/open-next/src/adapters/cache.ts:493]: It seems to be in weird scenarios, like when sustaining attacks with weird things on the end of URLs (\ is common).

All in all, not too much of an issue, I can ignore on our side, but I doubt we are the only website on the internet to get attacked by bots.

Comment (dkershner6) [reply packages/open-next/src/adapters/cache.ts:493]: I also think `warn` is appropriate.  Something weird happened, but no need to get out of bed in the middle of the night (unlike error).

**Review by khuezy — APPROVED** [review]

File: packages/open-next/src/adapters/cache.ts (hunk 1 above)
Comment (khuezy) [reply packages/open-next/src/adapters/cache.ts:493]: Is it a denial of wallet attack? Is the S3 LIST adding up on your account?

Comment (dkershner6) [reply packages/open-next/src/adapters/cache.ts:493]: Seems more like probing for now, but yes, I would assume it is to try and throw a wrench into some database call or something.

**Comment by dkershner6** [comment]
I added one more warn.  This error log is hit during the same flow after a mocked response and null coalesce.

File: packages/open-next/src/adapters/cache.ts (hunk 1 above)
Comment (conico974) [reply packages/open-next/src/adapters/cache.ts:493]: We should probably modify the `awslogger` that we pass to S3 as well.
https://github.com/sst/open-next/blob/e32af92b30738a52066208b8ea39a71fdb3be67f/packages/open-next/src/adapters/server-adapter.ts#L35

Comment (dkershner6) [reply packages/open-next/src/adapters/cache.ts:493]: > We should probably modify the `awslogger` that we pass to S3 as well.
> 
> https://github.com/sst/open-next/blob/e32af92b30738a52066208b8ea39a71fdb3be67f/packages/open-next/src/adapters/server-adapter.ts#L35

Done.

**Comment by dkershner6** [comment]
Added an interface for downplaying AWS SDK errors, with just the one for now, at @conico974 's request.

**Pull request merged**